frontend/src/component/admin/users/AccessOverview/AccessOverview.tsx

```diff
@@ -104,19 +106,24 @@ export const AccessOverview = () => {
                 </PageHeader>
             }
         >
-            <StyledTitle>
-                Root permissions for role {rootRole?.name}
-            </StyledTitle>
-            <AccessOverviewTable permissions={overview?.root ?? []} />
-            <StyledTitle>
-                Project permissions for project {project} with project roles [
-                {projectRoles?.map((role: any) => role.name).join(', ')}]
-            </StyledTitle>
-            <AccessOverviewTable permissions={overview?.project ?? []} />
-            <StyledTitle>
-                Environment permissions for environment {environment}
-            </StyledTitle>
-            <AccessOverviewTable permissions={overview?.environment ?? []} />
+            <StyledAccessOverviewContainer>
+                <AccessOverviewAccordion permissions={overview?.root ?? []}>
+                    Root permissions for role {rootRole?.name}
+                </AccessOverviewAccordion>
+                <AccessOverviewAccordion permissions={overview?.project ?? []}>
+                    Project permissions
+                    {project
+                        ? ` for project ${project}${projectRoles?.length ? ` with project role${projectRoles.length !== 1 ? 's' : ''} ${projectRoles?.map((role: any) => role.name).join(', ')}` : ''}`
+                        : ''}
```

Comment:
I really like this, but maybe lowercase? It looks weird to have some camel case some lower case, but on the flip side the name is the name... so it might be ok.
```suggestion
                    {project
                        ? ` for project ${project}${projectRoles?.length ? ` with project role${projectRoles.length !== 1 ? 's' : ''} ${projectRoles?.map((role: any) => role.name.lowercase()).join(', ')}` : ''}`
                        : ''}
```

Reply:
I'm not sure I agree. We're not doing the same with the project and environment. If you have roles like "Role A", "Role B", "Role C", or some other initials, it might look weird.

If you'd like a better visually hierarchal separation we can try adding a colon (:). Something like:
"with project roles: Role A, Role B, Role C".